### PR TITLE
Removes "thrall" prefix from thrall names and gives them a speech effect

### LINKED ISSUE #8640: Cloned thralls still have "thrall" prefix
### Labels

[MINOR]

### Description

After cloning a thrall when they are dead, they still have "thrall" before their name

### Steps to reproduce

1) Kill a thrall or find them dead
2) Scan and clone them
3) Have them pop out with "thrall" before their name despite being a human now

### Expected Behavior

Not having thrall prefix as it may confuse people

### Additional Information & Screenshots

Reported by: Nova_Hero
On server: Goonstation 1 Classic: Heisenbee
Round log date: 2022-05-19-00-56
Reported on: 2022-05-19 01:57:45

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[BALANCE] [GAMEMODES] [ADD TO WIKI] [BUG]
## About the PR 
Removes the eternally buggy "thrall" prefix from vampiric thralls, and instead gives them a speech effect causing their speech verb to be "gurgles", removes any ending punctuation from their messages and adds "..." to the end.
So `Yes!` becomes 
image
and `WHAT?!` becomes
image
Fixes #8640

## Why's this needed? 
Suggested here: https://forum.ss13.co/showthread.php?tid=19764
Makes it a bit more subtle, while still obvious when you speak.

## Changelog

```changelog
(u)LeahTheTech
(*)Vampiric thralls no longer have "thrall" added to their name, and instead have a gurgling speech effect.
```

### CODE REVIEW

**Comment by t-toasted** [comment]
Finally, someone has coded this. Really really good for RP, since you have to pretend not to notice the 'thrall' thing at the beginning, and it's still very obvious even without it. Will be good for classic too so they don't simply get validhunted for existing with their name.

**Comment by queen-mab** [comment]
thank u for the pr, i made the suggestion on forums so i hope it get merged soon! i think we need this especially on rp

File: code/modules/medical/genetics/bioEffects/speech.dm [hunk 1]
```diff
@@ -803,3 +803,27 @@
...
+	probability = 0
+	occur_in_genepools = 0
+	scanner_visibility = 0
+	can_research = 0
+	can_make_injector = 0
+	can_copy = 0
+	can_reclaim = 0
+	can_scramble = 0
+	curable_by_mutadone = 0
+	acceptable_in_mutini = 0
```
Comment (Tarmunora) [inline code/modules/medical/genetics/bioEffects/speech.dm:824]:
```suggestion
	acceptable_in_mutini = 1
```

**Review by Tarmunora — APPROVED** [review]

File: code/procs/mobprocs/chatprocs.dm [hunk 2]
```diff
@@ -449,7 +449,7 @@
 		if (src.bioHolder.HasEffect("accent_comic"))
 			font_accent = "Comic Sans MS"
 
-		if (src.bioHolder && src.bioHolder.genetic_stability < 50)
```
Comment (ZeWaka) [inline code/procs/mobprocs/chatprocs.dm:452]: why remove the bioholder check here?

Comment (Tarmunora) [reply code/procs/mobprocs/chatprocs.dm:452]: already checked on like 440

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**